src/core/Table/filters/BaseFilter.test.tsx

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
It is the same as FilterButtonBar so I don't think it is worth having duplicate tests. Better test filter icon click as clicking it also triggered sorting before.

Reply:
removed it, and I added your filter icon test